ResearchKit/Localized/es.lproj/ResearchKit.strings

```diff
@@ -204,6 +204,7 @@
 "BUTTON_APPLY" = "Aplicar";
 "BUTTON_DISAGREE" = "No acepto";
 "BUTTON_DONE" = "OK";
+"COMPLETION_BUTTON_DONE" = "LISTO";
```

Comment:
Linguistic nitpick: none of the other button actions is all-caps (except `OK` because that one is an odd case). I feel this would be `Listo`, otherwise it will look too prominent compared to other actions.

Reply:
@lopsae Just FYI, I've pushed for a change and was allowed to set it to "Listo" instead.